config/webpack/webpack.config.js

```diff
@@ -170,6 +173,14 @@ const webpackConfig = (options, env, argv) => {
   });
 };
 
+const basepathConfig = {
```

Comment:
You can just add the plugin to the devConfig plugin block instead of merging agian.